product_dimension/views/product_view.xml

```diff
@@ -4,32 +4,28 @@
     <record model="ir.ui.view" id="product_normal_form_view">
       <field name="name">product_normal_form_view</field>
       <field name="model">product.product</field>
-      <field name="inherit_id" ref="product.product_normal_form_view"/>
+      <field name="inherit_id" ref="product.product_variant_easy_edit_view"/>
       <field name="arch" type="xml">
-        <xpath expr="//page/group[@name='inventory']" position="inside">
-          <group name="dimensions">
+        <xpath expr="//label[@for='volume']" position="before">
             <field name="dimensional_uom_id"/>
             <field name="length" attrs="{'readonly':[('type','=','service')]}"/>
             <field name="height" attrs="{'readonly':[('type','=','service')]}"/>
             <field name="width" attrs="{'readonly':[('type','=','service')]}"/>
-          </group>
         </xpath>
       </field>
     </record>
 
     <record model="ir.ui.view" id="product_template_only_form_view">
-      <field name="name">product_template_form_view</field>
+      <field name="name">product.template with properties and dimensions</field>
       <field name="model">product.template</field>
-      <field name="inherit_id" ref="product.product_template_only_form_view"/>
+      <field name="inherit_id" ref="stock.view_template_property_form"/>
```

Comment:
`product` is not an app (in the odoo manifest sense), so IMO is not a big deal having modules depend on "real apps" (like `sale`, `purchase`, or `stock`). If you install only the `product` module, do you even have a menuitem to access them? I'm not sure!